domains/core/ui-design-system/src/sprinkles.css.ts

```diff
@@ -0,0 +1,81 @@
+import { defineProperties, createSprinkles } from "@vanilla-extract/sprinkles";
+
+import { theme } from "./theme.css";
+
+const responsiveProperties = defineProperties({
+  conditions: {
+    default: {},
+    desktop: {
+      "@media": "screen and (min-width: 1024px)",
+    },
+    tablet: {
+      "@media": "screen and (max-width: 1024px)",
+    },
+    mobile: {
+      "@media": "screen and (max-width: 640px)",
+    },
+    hover: { selector: "&:hover" },
+    focus: { selector: "&:focus" },
+    active: { selector: "&:active" },
+  },
+  defaultCondition: "default",
+  responsiveArray: ["desktop", "tablet", "mobile"],
+  properties: {
+    position: ["absolute", "relative", "fixed"],
+    display: ["none", "block", "inline", "inline-block", "flex", "inline-flex"],
+    alignItems: ["stretch", "flex-start", "center", "flex-end", "baseline"],
+    justifyContent: ["flex-start", "center", "flex-end", "space-between"],
+    flexDirection: ["row", "column", "row-reverse", "column-reverse"],
+    opacity: [0, 1],
+  },
```

Comment:
```suggestion
  },
  shorthands: {
    p: [
      'paddingTop',
      'paddingBottom',
      'paddingLeft',
      'paddingRight'
    ],
    pt: ['paddingTop'],
    pb: ['paddingBottom'],
    pl: ['paddingLeft'],
    pr: ['paddingRight'],
    px: ['paddingLeft', 'paddingRight'],
    py: ['paddingTop', 'paddingBottom']
  }
```

padding, margin 정도는 full 단축키 사용이 어떠하신지.

Reply:
For this, it is necessary to define properties for margin and padding. However, since our current design system does not have spacing definitions, we are unable to use shorthand functionality.

Reply:
Later.